src/item.cpp

```diff
@@ -10884,8 +10884,11 @@ bool item::is_upgrade() const
 
 int item::get_min_str() const
 {
+    Character &p = get_player_character();
```

Comment:
May as well mark you're not modifying it.
```suggestion
    const Character &p = get_player_character();
```